assets/css/style.css

```diff
@@ -788,3 +788,11 @@ form{
       transform: translateY(25%);
    }
   }
+
+  /* box shadow for profile picture */
+  .rounded-circle{
+    box-shadow: 10px 10px 5px #ccc;
```

Comment:
keep the x values to be zero and reduce the opacity of shadow by using rgb

Reply:
Okay, I have made another commit